lib/internal/streams/readable.js

```diff
@@ -418,34 +435,16 @@ function readableAddChunkUnshiftByteMode(stream, state, chunk, encoding) {
 }
 
 function readableAddChunkUnshiftObjectMode(stream, state, chunk) {
-  if (chunk === null) {
-    state[kState] &= ~kReading;
-    onEofChunk(stream, state);
-
-    return false;
-  }
-
   return readableAddChunkUnshiftValue(stream, state, chunk);
 }
 
 function readableAddChunkUnshiftValue(stream, state, chunk) {
```

Comment:
This function is now just 2 lines you can move the function content to the callers

Reply:
Done